lib/logstash/inputs/elasticsearch/aggregation.rb

```diff
@@ -16,7 +16,7 @@ def initialize(client, plugin)
           @query = @plugin_params["query"]
           @retries = @plugin_params["retries"]
           @agg_options = {
-            :index => @index,
+            :index => @plugin_params["index"],
             :size  => 0
```

Comment:
Good catch @kaisecheng!
How about size?

Reply:
The `size` should still be `0`, because this is specifically an _aggregations-only_ query and we don't need query results.